src/lib/timedateutils.ts

```diff
@@ -0,0 +1,26 @@
+export interface FormatDateProps {
+  date: Date;
+  variant?: string;
+}
+
+export function formatDate({ date, variant }: FormatDateProps): string {
+  const day = String(date.getDate()).padStart(2, '0');
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const year = date.getFullYear();
+  const hours = String(date.getHours()).padStart(2, '0');
+  const minutes = String(date.getMinutes()).padStart(2, '0');
+
+  if (variant === 'no-hours') {
+    return `${day}.${month}.${year}`;
+  } else {
+    return `${day}.${month}.${year} ${hours}:${minutes}`;
+  }
+}
+
```

Comment:
I think we should consider looking into a library such as https://momentjs.com/ if the need for different time util functions grows in the future

Reply:
Agree. I'm still quite unsure what our needs for manipulating times and dates will be though. On database level we have right now only start and end times for coures, which is quite little information. We don't have for example times of individual teaching instances within the timeframes. Probably could make decisions as a group later when we know what we exactly need.